internal/services/executor/driver/docker_test.go

```diff
@@ -450,4 +450,96 @@ func TestDockerPod(t *testing.T) {
 			t.Fatalf("unexpected exit code: %d", code)
 		}
 	})
+
+	t.Run("test pod with two tmpfs volumes with size limit", func(t *testing.T) {
+		pod, err := d.NewPod(ctx, &PodConfig{
+			ID:     uuid.NewV4().String(),
+			TaskID: uuid.NewV4().String(),
+			Containers: []*ContainerConfig{
+				&ContainerConfig{
+					Cmd:   []string{"cat"},
+					Image: "busybox",
+					Volumes: []Volume{
+						{
+							Path: "/mnt/vol1",
+							TmpFS: &VolumeTmpFS{
+								Size: 1024 * 1024,
+							},
+						},
+						{
+							Path: "/mnt/vol2",
+							TmpFS: &VolumeTmpFS{
+								Size: 1024 * 1024,
+							},
+						},
+					},
+				},
+			},
+			InitVolumeDir: "/tmp/agola",
+		}, ioutil.Discard)
+		if err != nil {
+			t.Fatalf("unexpected err: %v", err)
+		}
+		defer func() { _ = pod.Remove(ctx) }()
+
+		ce, err := pod.Exec(ctx, &ExecConfig{
+			Cmd: []string{"sh", "-c", "if [ $(grep /mnt/vol1 /proc/mounts | grep -c size=1024k) -ne 1 || $(grep /mnt/vol2 /proc/mounts | grep -c size=1024k) -ne 1 ]; then exit 1; fi"},
```

Comment:
Unfortunately this script has a syntax error but will return 0 anyway (the test passes also without the fix). 

`||` isn't a valid option for `[`. You could change it to `-o` or use `[[` that accepts both `||` and `-o`.